vignettes/train-with-tensorflow/train-with-tensorflow.Rmd

```diff
@@ -52,17 +52,19 @@ By using Azure Machine Learning Compute (AmlCompute), a managed service, data sc
 You may need to wait a few minutes for your compute cluster to be provisioned if it doesn't already exist.
 
 ```{r create_cluster, eval=FALSE}
-cluster_name <- "rcluster"
+cluster_name <- "gpucluster"
```

Comment:
are we planning to make same change for samples as well?

Reply:
yeah, I will do so